examples/data-objects/canvas/test/canvas.test.ts

```diff
@@ -55,17 +55,26 @@ describe("canvas", () => {
 				}
 				return context.getImageData(0, 0, width, height).data;
 			});
-			if (imgs[0].length == 0) {
+			const firstImg = imgs[0];
```

Comment:
I just made a change here that will conflict - suggest just disabling the rule for the package for now.